### ISSUE TITLE
How to save contact?

### ISSUE THREAD

**Issue by brianjmurrell** [BelledonneCommunications/linphone-android#1279]
Given that I have received a SIP SIMPLE message from a number not in my device's contact list(s), [how] can I add that number as a contact from linphone-android (on the kotlin_rewrite branch)?

Ideally linphone-android initiates the contact being added with (one of) the contact app(s) installed on the phone.

**Comment by Viish** [comment]
Long press one of the received message and click on "Add to contacts". 
You will be sent to the contacts list and from there either select an existing contact to add the SIP address to it or create a new one.

**Comment by brianjmurrell** [comment]
I see.  Yes, that does seem to work.

Maybe it's just me, but it would seem more intuitive to add a contact from the conversation list, not from a message in a conversation.  It's at the conversation list where you most notice the lack of any information for a contact, and only see a phone number, for a SIP SIMPLE conversation for example.

**Comment by Viish** [comment]
Yes, but that would only work for one to one conversations, not groups.

Also the long press on list items is already used for entering edition mode and pre-selecting the item.

**Comment by brianjmurrell** [comment]
> Yes, but that would only work for one to one conversations, not groups.

I did consider that, and in such a case, (group conversations) there is no concept of a _contact_, so my suggestion there is rather than having that add a contact, it let's one name the group conversation.

I did realize that long-press there is already bound to some functionality but I haven't quite gathered what, other than being able to delete multiple conversations in a batch.

> entering edition mode

What is "edition mode"?
